api/v1alpha1/clusterqueue_types.go

```diff
@@ -156,6 +157,10 @@ type QueueingStrategy string
 const (
 	// StrictFIFO means that workloads are sorted strictly by creation time.
```

Comment:
clarify: To preserve strict FIFO semantics, workloads that can't be admitted will block admitting newer workloads that can.

Reply:
Done